extensions/2.0/Khronos/KHR_materials_variants/schema/glTF.KHR_materials_variants.schema.json

```diff
@@ -3,7 +3,7 @@
     "title": "KHR_materials_variants glTF extension",
     "type": "object",
     "description": "glTF extension that defines a material variations for mesh primivites",
-    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "allOf": [ { "$ref": "../../glTFProperty.schema.json" } ],
```

Comment:
By convention we don't add this `../../` relative path to the official spec.  My gltf-vscode project has a script (`util/importAll.sh` with [docs](https://github.com/AnalyticalGraphicsInc/gltf-vscode/tree/master/util)) that adds these just to the gltf-vscode project internally.

Reply:
Makes sense, will fix